specification/paths/ServiceRates.json

```diff
@@ -0,0 +1,53 @@
+{
+  "post": {
+    "tags": [
+      "Service rates"
+    ],
+    "summary": "Get service rates for a shipment",
+    "description": "This endpoint retrieves service rates of the carrier based on the supplied shipment.",
+    "parameters": [
+      {
+        "in": "body",
+        "name": "shipment",
+        "description": "The shipment object to be created.",
+        "required": true,
+        "schema": {
+          "type": "object",
+          "required": [
+            "data"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "data": {
+              "$ref": "#/definitions/Shipment"
```

Comment:
The story says we need to make the attributes.service.code optional. Also the shipment will not have a `myparcelcom_shipment_id` yet, since we request this info in the new shipment form.

It's annoying this project is still `OpenApi 2.0` and not `OpenApi 3.0` like our api-specification, so we cannot use `allOf` to merge JSON. Let's make a new `BaseShipment.json` definition with the same fields, but nothing required. At some point we will refactor this when we switch to OpenApi 3.